packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2078,6 +2078,16 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             const summaryRefSeqNum = this.deltaManager.lastSequenceNumber;
             const message = `Summary @${summaryRefSeqNum}:${this.deltaManager.minimumSequenceNumber}`;
 
+            // We should be here is we haven't processed be here. If we are of if the last message's sequence number
+            // doesn't match the last processed sequence number, log an error.
+            if (summaryRefSeqNum !== this.deltaManager.lastMessage?.sequenceNumber) {
+                summaryLogger.sendErrorEvent({
+                    eventName: "LastSequenceMismatch",
+                    lastSequenceNumber: summaryRefSeqNum,
```

Comment:
your will get the delta manager columns for free: https://github.com/microsoft/FluidFramework/blob/main/packages/loader/container-loader/src/container.ts#L562L573
lastSequenceNumber is misleading. i also think this should just be summaryRefSeqNum